thegreatknowledgeheist/models/base_model_config.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+
+from thegreatknowledgeheist.dataset.data_config import DataConfig
+from thegreatknowledgeheist.types import TASK
+
+
+@dataclass
+class BaseModelConfig:
```

Comment:
Możnaby tutaj pewnie zadbać o to, żeby jako model_name nie zostały przekazane jakieś śmieci, np. przez użycie Enum albo typing.Literal, ale to może być już trochę overengineering.